.github/workflows/upload_component.yml

```diff
@@ -0,0 +1,21 @@
+name: Push components to Espressif Component Service
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  upload_components:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          submodules: "recursive"
+
+      - name: Upload components to component service
+        uses: espressif/github-actions/upload_components@master
+        with:
+          name: "libnmea"
+          namespace: "espressif"

```

Comment:
I think I'd prefer to keep this component under my personal namespace, not `espressif`. I'll check with @kumekay if creating such namespace is possible already.

Reply:
Yes, it's possible but manually at the moment. What namespace do you what for personal projects?

Reply:
`igrr`, if it's not taken :) Thanks in advance

Reply:
I've created the namespace (and empty component in it)

Reply:
Namespace updated to `igrr`. 
API token updated too, to reflect namespace change